src/main/resources/assets/goodgenerator/data/essentia.json

```diff
@@ -152,19 +152,19 @@
     },
     {
       "name": "Praecantatio",
-      "fuelValue": 36000,
+      "fuelValue": 30000,
       "category": "NORMAL",
       "consumeCeo": 0.0
     },
     {
       "name": "Auram",
-      "fuelValue": 42000,
+      "fuelValue": 76800,
```

Comment:
Why did Auram just randomly become so good? It has a lot of decent sources? I'm just curious about the thought process

Reply:
Auram's fluid consumption rate was increased, so that's kind of a compensation